### PR TITLE
Use RPC instead of exec.Command(...)

### LINKED ISSUE #152: Use RPC instead of calling ourselves
```
util/issue_commands.go:20:      cmdArgStr := fmt.Sprintf("planetmint-god tx dao reissue-rddl-proposal %s '%s' %s --from %s -y",
util/issue_commands.go:49:      cmdArgStr := fmt.Sprintf("planetmint-god tx dao reissue-rddl-result %s '%s' %s --from %s -y",
```

When this is called, `planetmint-god start` was already called...

### PR DESCRIPTION
**Pull request by jmastr**
// Closes #152

### CODE REVIEW

File: util/issue_commands.go [hunk 1]
```diff
@@ -1,59 +1,39 @@
...
-func InitRDDLReissuanceProcess(ctx sdk.Context, proposerAddress string, txUnsigned string, blockHeight int64) error {
+func buildSignBroadcastTx(ctx sdk.Context, sendingValidatorAddress string, msg sdk.Msg) (err error) {
+	logger := ctx.Logger()
+	addr := sdk.MustAccAddressFromBech32(sendingValidatorAddress)
+	txBytes, txJSON, err := lib.BuildAndSignTx(addr, msg)
+	if err != nil {
+		return
+	}
+	logger.Debug("REISSUE: tx: " + txJSON)
+	_, err = lib.BroadcastTx(txBytes)
```
Comment (eckelj) [inline util/issue_commands.go:18]: Is this call result in an async call or at least issued in a non-blocking way?
I ask that as if not, we will have difficulties debugging and analysing the workflows within the planetmint-go service.

Comment (jmastr) [reply util/issue_commands.go:18]: BroadcastMode is the default behavior: `BROADCAST_MODE_SYNC`

Comment (jmastr) [reply util/issue_commands.go:18]: and then it is a `http.Post`

Comment (jmastr) [reply util/issue_commands.go:18]: "each HTTP request is handled in it's own go routine" https://stackoverflow.com/a/52157993

**Review by eckelj — APPROVED** [review]

**Pull request merged**